app/forms/firewall-rules-create.tsx

```diff
@@ -182,8 +215,6 @@ export const CommonFields = ({ error, control }: CommonFieldsProps) => {
       <NameField name="name" control={control} />
       <DescriptionField name="description" control={control} />
 
-      <FormDivider />
-
       <NumberField
         name="priority"
```

Comment:
Any thoughts about moving this to the end? I think having it first gives it undue importance.

Reply:
Moved it to the bottom of the top section.